consensus/src/quorum_store/utils.rs

```diff
@@ -314,13 +319,30 @@ impl ProofQueue {
                         .observe((current_time.round() - expiration.round()) as f64);
                 }
             }
+            size = size - 1;
         }
         counters::EXPIRED_PROOFS_WHEN_PULL.observe(num_expired_but_not_committed as f64);
         counters::BLOCK_SIZE_WHEN_PULL.observe(cur_txns as f64);
         counters::BLOCK_BYTES_WHEN_PULL.observe(cur_bytes as f64);
         counters::PROOF_SIZE_WHEN_PULL.observe(ret.len() as f64);
```

Comment:
Should we move this into the if block below, so we only count when we return a "good" result?

Reply:
fixed